specification/components/schemas/AppointmentSlotBundle.yaml

```diff
@@ -66,9 +66,15 @@ properties:
           example: '_baseUrl_/STU3/v1/Slot?schedule.actor%3AHealthcareService=11010&appointmentType=ROUTINE&_count=10&status=free&_include=Slot%3Aschedule&page=1'
   entry:
     type: array
+    description: |
+      | Field                  | Cardinality | Notes                                      |
+      | ---------------------- | ----------- | ------------------------------------------ |
+      | `Slot`                 | `1..*`      |                                            |
+      | `Schedule`             | `1..*`      |                                            |
     items:
       anyOf:
-        - title: |
+        - title: Slot
```

Comment:
What's the difference between title and description here? Are they displayed differently when rendered by the NHS developer site?

Reply:
Yeah title is bold, description is normal font. No other difference (currently)